aspen/http/resource.py

```diff
@@ -7,25 +8,18 @@ class Static(object):
     """
 
     def __init__(self, request_processor, fspath, raw, media_type):
-        self.request_processor = request_processor
+        assert type(raw) is bytes  # sanity check
         self.raw = raw
         self.media_type = media_type
         if media_type == 'application/json':
-            self.media_type = self.request_processor.media_type_json
+            self.media_type = request_processor.media_type_json
+        if self.media_type.startswith('text/'):  # XXX this is bad
```

Comment:
[Indeed](https://github.com/AspenWeb/aspen.py/blob/0aff41b1ee78db9cb7a4aa04c6c1e34580211b22/aspen/request_processor/__init__.py#L27)! I had assumed the default was `utf-8`. :)

In that case maybe we should just drop `charset_static` entirely, and let users implement their own algorithm function for varying the charset of static files if they want.